cfme/tests/automate/test_domain.py

```diff
@@ -79,9 +79,14 @@ def test_domain_edit_enabled(domain, appliance):
     view = navigate_to(domain, 'Details')
     assert 'Disabled' in view.title.text
 
+    @request.addfinalizer
+    def _finalize():
+        with update(domain):
+            domain.enabled = True
```

Comment:
Optional. It would be better to set the previous state somewhere and set it back in the finalizer. Your code is assuming it was True.

Reply:
Since the test is asserting it wasn't disabled before disabling it, I'd agree its optional.  As a note for future test development @dgaikwad, I would wrap this behavior into the domain fixture itself.